bazel/third_party/glog/0002-mark-override.patch

```diff
@@ -0,0 +1,36 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
What is this file used for? Can we delete it?

Reply:
This solved a noisy compiler warning. I suggest applying it. The final user could use their provided glog. It's harmless.